x-pack/plugins/ml/public/application/services/results_service/result_service_rx.ts

```diff
@@ -347,9 +348,10 @@ export function resultsServiceRxProvider(mlApiServices: MlApiServices) {
       jobIds: string[],
       criteriaFields: CriteriaField[],
       threshold: any,
-      earliestMs: number,
-      latestMs: number,
-      maxResults: number | undefined
+      earliestMs: number | null,
+      latestMs: number | null,
+      maxResults: number | undefined,
+      functionDescription?: string | undefined
```

Comment:
As noted above, you should be able to remove ` | undefined` because of the `?`.

Reply:
Updated here https://github.com/elastic/kibana/pull/81662/commits/64ecedbd1dccaa5b916c4e81538729a17dfb3afb